### PR TITLE
Categorize universe data articles

### LINKED ISSUE #1553: Pedia Categories Monster Types and Ship Design Empty From Intro Screen
When starting FreeOrion, the pedia from the intro screen has Monster Types and Ship Design categories listed in the index. These are empty until a game starts and the server sends the relevant info. They should thus probably be labelled as "[Game -", like Buildings or Ships, which are populated with game-specific info.

Alternative, these articles could be populated with info read from the default scripted stuff.

### PR DESCRIPTION
**Pull request by dbenage-cx**
Moves the '[Game - ***]', and both normal and monster ship designs pedia categories as sub-categories to a new top-level category.

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -3897,6 +3897,14 @@ ENC_SUITABILITY_REPORT_WHEEL_INTRO
 Suitability is determined by distance from the preferred type of [[encyclopedia ENVIRONMENT_TITLE]] in this wheel:
 '''
 
+ENC_EMPIRE_DATA
+[Current Game - Universe]
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Why is it "empire data" if it's about the universe?

File: UI/EncyclopediaDetailPanel.cpp [hunk 2]
```diff
@@ -137,11 +137,26 @@ namespace {
...
+            "ENC_MONSTER",
+            "ENC_MONSTER_TYPE",
+            "ENC_FLEET",
+            "ENC_PLANET",
+            "ENC_BUILDING",
+            "ENC_SYSTEM",
+            "ENC_FIELD",
+            "ENC_GRAPH",
+            "ENC_GALAXY_SETUP",
+            "ENC_HOMEWORLDS"
```
Comment (geoffthemedio) [inline UI/EncyclopediaDetailPanel.cpp]: categorizing is fine, but a lot of this stuff seems like it's useful enough to have at the top tier listing of articles... for quick access

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -3897,6 +3897,14 @@ ENC_SUITABILITY_REPORT_WHEEL_INTRO
...
+
+ENC_EMPIRE_DATA_DESC
+'''The latest information avialable on the current universe.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: lose "the latest" and "available". maybe "information about the current universe"

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -3897,6 +3897,14 @@ ENC_SUITABILITY_REPORT_WHEEL_INTRO
...
+
+Not all objects or types are listed here, only those known to your empire.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: don't refer to the player in second person (ie. as "you")

File: UI/EncyclopediaDetailPanel.cpp (hunk 2 above)
Comment (dbenage-cx) [reply UI/EncyclopediaDetailPanel.cpp]: I've posted this back to [forum discussion](http://www.freeorion.org/forum/viewtopic.php?f=6&t=10343&start=15#p87920), will mark this WIP.

File: default/stringtables/en.txt (hunk 1 above)
Comment (dbenage-cx) [reply default/stringtables/en.txt]: Changed to `ENC_EMPIRE_UNIVERSE_DATA` to reflect a limited view of the universe.

**Comment by Vezzra** [comment]
How important is this for 0.4.7? Unless it's kind of critical, I'd like to reassign it to the "optional for 0.4.7" milestone, so it won't hold up the release.

**Comment by geoffthemedio** [comment]
@dbenage-cx you've got about 4 work in progress tagged pull requests now. could you finish / clean up some of them so they're ready to be merged?

**Comment by dbenage-cx** [comment]
Rebased to master with additions for repurposed graph button per suggestion by Ouaz in forum.

**Comment by geoffthemedio** [comment]
I don't really see the point of this... To me, it just means more clicks are required to get to the information. There aren't so many items at the top level that extra categorization is needed...

**Comment by dbenage-cx** [comment]
Ok, thanks for update, closing as invalid.

**Comment by geoffthemedio** [comment]
others may disagree, though...

**Comment by dbenage-cx** [comment]
I've [posted](http://freeorion.org/forum/viewtopic.php?f=6&t=10714) a request for feedback.
Please correct me if I've misrepresented your stance.

**Pull request closed without merging**